bin/maintenance/update_backrefs.py

```diff
@@ -15,6 +15,7 @@
 # along with Indico; if not, see <http://www.gnu.org/licenses/>.
 
 from __future__ import unicode_literals
+from __future__ import print_function
```

Comment:
We merge imports from the same source -> 

    from __future__ import print_function, unicode_literals